projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss

```diff
@@ -30,6 +33,33 @@
       margin-left: 0;
     }
   }
+  
+  .extraction-menu, .anatomical-structures-menu {
+    margin: 0;
```

Comment:
All theses `margin: 0` is because you haven't fixed the underlying problem which is the `:not(hr)` and `*:first-child` are to general and needs to be made more specific i.e. `& > :not(hr)` and `& > :first-child`